app/scripts/models/auth_brokers/fx-sync-channel.js

```diff
@@ -40,7 +41,9 @@ define(function (require, exports, module) {
     commands: null,
 
     defaultCapabilities: _.extend({}, proto.defaultCapabilities, {
-      sendChangePasswordNotice: true
+      sendAfterSignInConfirmationPollNotice: false,
+      sendAfterSignUpConfirmationPollNotice: false,
```

Comment:
The above comment is not to mean that the semantics the content server uses are not totally messed up and confusing. The _are both_. I'd love help renaming these methods/concepts internally, every person that comes into the project is confused by the names & corresponding URL scheme.